management-core/mysqldb.dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM mysql/mysql-server
```

Comment:
This is because I did not fully understand what piece do what, right now I have an idea, just check official mysql docker and changed it, can you tell me this is correct?

Reply:
Yes please, according docker best practices, we should go to small images and official ones